src/main/kotlin/no/nav/syfo/melding/domain/MeldingTilBehandler.kt

```diff
@@ -56,18 +56,24 @@ data class MeldingTilBehandler(
             opprinneligMelding: MeldingTilBehandler,
             veilederIdent: String,
             document: List<DocumentComponentDTO>
-        ): MeldingTilBehandler = create(
-            type = MeldingType.FORESPORSEL_PASIENT_PAMINNELSE,
-            conversationRef = opprinneligMelding.conversationRef,
-            parentRef = opprinneligMelding.uuid,
-            personIdent = opprinneligMelding.arbeidstakerPersonIdent,
-            behandlerPersonIdent = opprinneligMelding.behandlerPersonIdent,
-            behandlerNavn = opprinneligMelding.behandlerNavn,
-            behandlerRef = opprinneligMelding.behandlerRef,
-            tekst = "",
-            document = document,
-            veilederIdent = veilederIdent,
-        )
+        ): MeldingTilBehandler {
+            if (!opprinneligMelding.kanHaPaminnelse()) {
+                throw IllegalArgumentException("Kan ikke opprette påminnelse for melding av type ${opprinneligMelding.type}")
+            }
```

Comment:
Trenger vi denne sjekken, egentlig? Det vil vel aldri oppstå en mulighet for å sende påminnelse på de andre meldingstypene, siden det lages ingen oppgaver for de via cronjobben. 

Det er kanskje mest for safety measures?